src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,19 @@
+import { CardBody } from 'react-bootstrap';
+import Card from 'react-bootstrap/Card';
+
+export default function ArticleCard( { article } ) {
+    return (
+        <>
```

Comment:
This empty fragment seems unnecessary as you already have a Card component wrapping everything within here